content/constants.js

```diff
@@ -1,12 +1,25 @@
 const path = require("path");
+const fs = require("fs");
 const { VALID_LOCALES } = require("../libs/constants");
 
 require("dotenv").config({
   path: path.join(__dirname, "..", process.env.ENV_FILE || ".env"),
 });
 
-const CONTENT_ROOT = process.env.CONTENT_ROOT;
-console.assert(CONTENT_ROOT, "Env var CONTENT_ROOT must be set");
+const attemptedContentRoot = process.env.CONTENT_ROOT;
+let CONTENT_ROOT;
+if (!attemptedContentRoot) {
+  throw "Env var CONTENT_ROOT must be set";
+}
+if (!fs.existsSync(attemptedContentRoot)) {
```

Comment:
I can't remember exactly why but looking at how `server/index.js` used to do, it makes me think this should be 
```suggestion
if (!fs.existsSync(path.resolve(attemptedContentRoot))) {
```